array_shift.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+function insertShiftArray(arr, n) {
+    let newArr = [];
+    let middle = Math.ceil(arr.length/2);
```

Comment:
"Without utilizing any of the built-in methods available to your language" <- Be careful about this